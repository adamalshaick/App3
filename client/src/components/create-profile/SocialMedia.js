import React from "react";
import InputGroup from "../common/InputGroup";

const SocialMedia = ({
  twitter,
  facebook,
  linkedin,
  youtube,
  instagram,
  onChange,
  errors
}) => {
  return (
    <section>
      <InputGroup
        id="twitter"
        placeholder="Twitter Profile URL"
        name="twitter"
        icon="fab fa-twitter"
        value={twitter}
        onChange={onChange}
        error={errors.twitter}
      />

      <InputGroup
        id="facebook"
        placeholder="Facebook Page URL"
        name="facebook"
        icon="fab fa-facebook"
        value={facebook}
        onChange={onChange}
        error={errors.facebook}
      />

      <InputGroup
        id="linkedin"
        placeholder="Linkedin Profile URL"
        name="linkedin"
        icon="fab fa-linkedin"
        value={linkedin}
        onChange={onChange}
        error={errors.linkedin}
      />

      <InputGroup
        id="youtube"
        placeholder="YouTube Channel URL"
        name="youtube"
        icon="fab fa-youtube"
        value={youtube}
        onChange={onChange}
        error={errors.youtube}
      />

      <InputGroup
        id="instagram"
        placeholder="Instagram Page URL"
        name="instagram"
        icon="fab fa-instagram"
        value={instagram}
        onChange={onChange}
        error={errors.instagram}
      />
    </section>
  );
};

export default SocialMedia;
