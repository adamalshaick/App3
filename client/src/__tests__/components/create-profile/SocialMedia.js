import React from "react";
import { shallow } from "enzyme";
import SocialMedia from "../../../components/create-profile/SocialMedia";
import toJson from "enzyme-to-json";

const Props = {
  errors: {}
};

const wrapper = shallow(<SocialMedia {...Props} />);

describe("render component", () => {
  it("renders component", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
