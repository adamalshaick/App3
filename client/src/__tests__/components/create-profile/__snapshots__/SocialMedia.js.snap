// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component renders component 1`] = `
<section>
  <InputGroup
    icon="fab fa-twitter"
    id="twitter"
    name="twitter"
    placeholder="Twitter Profile URL"
    type="text"
  />
  <InputGroup
    icon="fab fa-facebook"
    id="facebook"
    name="facebook"
    placeholder="Facebook Page URL"
    type="text"
  />
  <InputGroup
    icon="fab fa-linkedin"
    id="linkedin"
    name="linkedin"
    placeholder="Linkedin Profile URL"
    type="text"
  />
  <InputGroup
    icon="fab fa-youtube"
    id="youtube"
    name="youtube"
    placeholder="YouTube Channel URL"
    type="text"
  />
  <InputGroup
    icon="fab fa-instagram"
    id="instagram"
    name="instagram"
    placeholder="Instagram Page URL"
    type="text"
  />
</section>
`;
